### ISSUE TITLE
Android Auto cannot connect

### ISSUE THREAD

**Issue by micahsoftdotexe** [GrapheneOS/os-issue-tracker#3463]
Greetings
I have recently installed graphene os and the Android Auto app (version 11.4.640814-release) on my Pixel 7 pro and I tried to connect to my radio (via usb) but could not get the radio working (Pioneer DMH-1500NEX). These are the following steps that I have tried:

- Connected phone to android auto enabled radio using usb
- Notification appeared saying that I needed to set up android auto in `Settings > Apps > Sandboxed Google Play > Android Auto`
- I did not see `Sandboxed Google Play` in my apps even though I have all dependencies for Android Auto installed so I went to `Settings > Apps > Android Auto > Additional settings in the app`
- Went to `Connect A Car` but it did not think usb was connected to car
It appears as if the system recognizes the Android Auto enabled radio while Android Auto itself does not. Did I set something up wrong and did I not give proper permissions to the app or is this an issue?

**Comment by muhomorr** [comment]
"Sandboxed Google Play" item is under the General category of the Apps screen in Settings app, it's not a part of list of installed apps.

**Comment by micahsoftdotexe** [comment]
Yes I do. I just found the correct menu. My eyes skipped over the clearly labeled `Sandboxed Google Play` button and I went to all apps to try to find it in the list of apps. Sorry for the inconvenience. It is working now!
